joint_limits/package.xml

```diff
@@ -0,0 +1,24 @@
+<package format="3">
+  <name>joint_limits</name>
+  <version>0.0.0</version>
+  <description>Interfaces for handling of joint limits for controllers or hardware.</description>
+
+  <maintainer email="bence.magyar.robotics@gmail.com">Bence Magyar</maintainer>
+  <maintainer email="denis@stogl.de">Denis Štogl</maintainer>
+
+  <license>Apache License 2.0</license>
+
+  <url type="website">https://github.com/ros-controls/ros2_control/wiki</url>
+  <url type="bugtracker">https://github.com/ros-controls/ros2_control/issues</url>
+  <url type="repository">https://github.com/ros-controls/ros2_control</url>
+
+  <buildtool_depend>ament_cmake</buildtool_depend>
+
+  <depend>rclcpp</depend>
+
+  <test_depend>ament_cmake_gtest</test_depend>
```

Comment:
Please add `<test_depend>launch_testing_ament_cmake</test_depend>` here. Then rosdep will install it before building.

Reply:
Thanks for this!